x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -410,39 +438,31 @@ const ProcessEventDotComponents = React.memo(
         type: 'userSelectedResolverNode',
         payload: {
           nodeId,
+          selectedProcessId: selfId,
         },
       });
-    }, [animationTarget, dispatch, nodeId]);
+      pushToQueryParams({ crumbId: selfEntityId, crumbEvent: 'all' });
+    }, [animationTarget, dispatch, nodeId, selfEntityId, pushToQueryParams, selfId]);
 
-    const handleRelatedEventRequest = useCallback(() => {
-      dispatch({
-        type: 'userRequestedRelatedEventData',
-        payload: event,
-      });
-    }, [dispatch, event]);
-
-    const handleRelatedAlertsRequest = useCallback(() => {
-      dispatch({
-        type: 'userSelectedRelatedAlerts',
-        payload: event,
-      });
-    }, [dispatch, event]);
     /**
      * Enumerates the stats for related events to display with the node as options,
      * generally in the form `number of related events in category` `category title`
      * e.g. "10 DNS", "230 File"
      */
-    const relatedEventOptions = useMemo(() => {
+
+    const [relatedEventOptions, grandTotal] = useMemo(() => {
       const relatedStatsList = [];
 
       if (!relatedEventsStats) {
         // Return an empty set of options if there are no stats to report
-        return [];
+        return [[], 0];
       }
+      let runningTotal = 0;
       // If we have entries to show, map them into options to display in the selectable list
       for (const category in relatedEventsStats.events.byCategory) {
         if (Object.hasOwnProperty.call(relatedEventsStats.events.byCategory, category)) {
           const total = relatedEventsStats.events.byCategory[category];
+          runningTotal += total;
```

Comment:
Just checking to see if we could use the total from the backend here as well.

Reply:
Same as above - good catch.